guides/v2.1/cloud/project/project-upgrade.md

```diff
@@ -13,137 +9,83 @@ functional_areas:
   - Cloud
   - Upgrade
 ---
+You can upgrade the core {{site.data.var.ee}} code base to version 2.1. If you need to upgrade from an older version than listed, you must upgrade to a supported version first.
 
-This information details how to upgrade {{site.data.var.ece}} from any version to 2.1.X.
+It is a best practice to verify the `ADMIN_EMAIL` variable, because it is required for upgrading and patching {{site.data.var.ece}}. See [Set environment and project variables]({{ page.baseurl }}/cloud/project/project-webint-basic.html#project-conf-env-var).
 
-When you upgrade {{site.data.var.ece}}, you also upgrade with patches and available hotfixes as part of the `magento-cloud-metapackage`. Make sure that
-your project root folder contains an `auth.json` file for authentication.
+{% include cloud/note-upgrade.md %}
 
-Our upgrades are Composer driven. For more information on Composer, see [Composer in Cloud]({{ page.baseurl }}/cloud/reference/cloud-composer.html).
+{% include cloud/note-ece-tools-package.md %}
 
-<div class="bs-callout bs-callout-warning" markdown="1">
-Always apply and test a patch your local system in an active branch. You can push and test in an Integration environment prior to deploying across all environments.
-</div>
+## Back up the database
 
-We recommend that you first back up the database of the system you are upgrading. Use the following steps to back up your integration, staging, and production systems.
+{% include cloud/backup-db.md %}
 
-## Verify or set the ADMIN_EMAIL variable {#variable}
-The environment variable `ADMIN_EMAIL` is required for upgrading and patching. This email is used for sending password reset requests and verified during when updating {{site.data.var.ece}}. See [[Set environment and project variables]({{ page.baseurl }}/cloud/project/project-webint-basic.html#project-conf-env-var).
+## Complete the upgrade
 
-## Back up the database {#backup-db}
-Back up your integration system database and code:
+1.  Change to your Magento base directory and set the upgrade version.
```

Comment:
Is this the same as my Magento root directory? The "base" term gives me pause for some reason. I'm not accustomed to seeing it and if it's not root, then I'm not sure what it is..